src/Workspaces/CSharp/Portable/Extensions/CastExpressionSyntaxExtensions.cs

```diff
@@ -213,6 +213,18 @@ private static bool PointerCastDefinitelyCantBeRemoved(CastExpressionSyntax cast
             return false;
         }
 
+        private static bool EnumCastDefinitelyCantBeRemoved(CastExpressionSyntax cast, ITypeSymbol expressionType)
```

Comment:
:memo: This name was picked to be consistent with a name for a similar method above it.